include/irr/asset/IDescriptorSet.h

```diff
@@ -143,9 +143,11 @@ class IEmulatedDescriptorSet
 			if (!_layout)
 				return;
 
-			const auto& bindings = _layout->getBindings();
+			auto bindings = _layout->getBindings();
```

Comment:
why do you get rid of my `const auto&` or `auto&`, we should be using it as often as possible to make sure accidental copy constructions don't happen.

Reply:
because its just pair of pointers and `const auto&` just holds rvalue retuned from `getBindings` because of lifetime expansion which seemed weird to me

Reply:
hmm ok, but most of the time we want to use `auto&` or `const auto&` because you can't see the type easily, I thought it was one of our containers.

Reply:
ofc, im not a c++ newbie, i care about copying